src/main/java/org/jenkinsci/plugins/codesonar/models/CodeSonarAlertFrequencies.java

```diff
@@ -0,0 +1,70 @@
+package org.jenkinsci.plugins.codesonar.models;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarAlertFrequencies {
```

Comment:
I think this class doesn't collect frequencies, rather it collects raw alert counts related to an analysis.  Probably a better name for this `CodeSonarAlertCounter`. If it were collecting frequencies the getters would return fractions, not whole numbers.

Also, I don't think this class is a "model" in the sense of it being used for deserializing JSON data.  It seems like this class doesn't belong in this sub-package.